services/QuillLMS/app/mailers/admin_dashboard_user_mailer.rb

```diff
@@ -57,4 +57,13 @@ def made_school_admin_change_school_email(user, admin_name, new_school, existing
     mail to: user.email, subject: "#{user.first_name}, you are now a Quill admin for #{new_school.name}"
   end
 
+  private def link_for_setting_password(role)
+    params = {
+      accountType: role,
+      adminFullName: @admin_name,
+      schoolName: @school_name
+    }
```

Comment:
Sounds good, I'll move it to the `UserMailer` class. I don't think we need to worry about the params being null because `.to_query` will just stringify them as null values, so when the frontend checks for their presence it won't find anything.